index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<link rel="stylesheet" type="text/css" href="styles.css">
+	<title>Home | Chandler Morrison</title>
+	<link href="https://fonts.googleapis.com/css?family=Inconsolata|Montserrat|Oxygen" rel="stylesheet">
+<ul class="NavBar">
+	<li><a href="index.html">Home</a></li>
+	<li><a href="bio.html">Bio</a></li>
+
+</ul>
+
+
+
+
+
+
+
+
+
+
+
+
+</head>
+
+<body background="images/Jacket.jpg">
```

Comment:
Try to keep any styling reserved for your CSS file. Inline styling can lead to maintenance problems down the road.